src/components/common/Header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
Vercell commerce uses navbar.tsx for this component's name

Reply:
Got that, Navbar.tsx already have a <header> in it. Just realized that now, thanks.